lib/bambora/bank/payment_profile_resource.rb

```diff
@@ -0,0 +1,78 @@
+# frozen_string_literal: true
+
+module Bambora
+  module Bank
+    ##
+    # For making requests to the /scripts/payment_profiles.asp endpoint
+    #
+    # @see https://help.na.bambora.com/hc/en-us/articles/115010346067-Secure-Payment-Profiles-Batch-Payments
+    class PaymentProfileResource
+      DEFAULT_VERSION = 1.0
+      DEFAULT_RESPONSE_FORMAT = 'QS'
+      attr_reader :client, :api_key, :sub_path, :version
+
+      ##
+      # Instantiate an interface to make requests against Bambora's Profiles API.
+      #
+      # @example
+      #
+      #   client = Bambora::WWWFormClient(base_url: '...', api_key: '...', merchant_id: '...')
+      #   profiles = Bambora::Bank::PaymentProfileResource(client: client)
+      #
+      #   # Start making requests ...
+      #
+      # @param client [Bambora::JSONClient] An instance of Bambora::JSONClient, used to make network requests.
+      # @param api_key [String] An API key for this endpoint. This is also known as the "Pass Code"
+      # @param version [String] The Service Version you are requesting from the server.
+      def initialize(client:, api_key:)
```

Comment:
Is `version` missing as a keyword argument here?

Reply:
It is!